aaad/quotas.py

```diff
@@ -42,8 +51,31 @@ def get_task_allocation(self, bucket_name):
             ''' Computes and returns total allocation across all tasks using the bucket '''
             if not self.quota_data['buckets'].get(bucket_name):
                 return None
-            allocation = { 'resources': {'cpus': 0, 'mem': 0 } }
-            # TODO - Implementation pending
+            allocation = { 'resources': {'cpus': 0.0, 'mem': 0.0, 'disk': 0.0 } }
+            if bucket_name:
+                allowed_names = []
+                for item in self.quota_data['names']:
```

Comment:
why look in `names`?
shouldn't this be looking at the `buckets.keys`?
something alone `if bucket_name in buckets.keys`?